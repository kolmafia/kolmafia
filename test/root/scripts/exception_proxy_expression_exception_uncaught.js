const { print } = require("kolmafia");

Effect.get("this effect does not exist"); // this will throw an exception
print("This code should not be reached.");