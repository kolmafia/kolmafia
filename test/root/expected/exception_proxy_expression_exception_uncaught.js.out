Script exception: Bad effect value: this effect does not exist